src/engine/bank4b.asm

```diff
@@ -842,127 +842,127 @@ _LoadOWObject:
-	dw TILESET_12A, SPRITE_ANIM_62, OWFRAMEGROUP_2F ; OWSPRITE_4D
-	dw TILESET_12B, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_GRYOUNGSTER
-	dw TILESET_12C, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_4F
-	dw TILESET_12D, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_50
-	dw TILESET_12E, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_51
-	dw TILESET_12F, SPRITE_ANIM_60, OWFRAMEGROUP_2D ; OWSPRITE_52
-	dw TILESET_130, SPRITE_ANIM_7C, OWFRAMEGROUP_23 ; OWSPRITE_53
-	dw TILESET_131, SPRITE_ANIM_62, OWFRAMEGROUP_2F ; OWSPRITE_GIRL
-	dw TILESET_132, SPRITE_ANIM_63, OWFRAMEGROUP_30 ; OWSPRITE_LASS1
-	dw TILESET_133, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_LASS2
-	dw TILESET_134, SPRITE_ANIM_65, OWFRAMEGROUP_32 ; OWSPRITE_LASS3
-	dw TILESET_135, SPRITE_ANIM_60, OWFRAMEGROUP_2D ; OWSPRITE_SWIMMER
-	dw TILESET_136, SPRITE_ANIM_60, OWFRAMEGROUP_2D ; OWSPRITE_GAL
-	dw TILESET_137, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_WOMAN
-	dw TILESET_138, SPRITE_ANIM_60, OWFRAMEGROUP_2D ; OWSPRITE_GRANNY
-	dw TILESET_139, SPRITE_ANIM_62, OWFRAMEGROUP_2F ; OWSPRITE_5C
-	dw TILESET_13A, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_GRLASS1
-	dw TILESET_13B, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_GRGAL
-	dw TILESET_13C, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_GRWOMAN
-	dw TILESET_13D, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_GRGRANNY
-	dw TILESET_13E, SPRITE_ANIM_5F, OWFRAMEGROUP_2C ; OWSPRITE_61
-	dw TILESET_13F, SPRITE_ANIM_63, OWFRAMEGROUP_30 ; OWSPRITE_62
-	dw TILESET_140, SPRITE_ANIM_7D, OWFRAMEGROUP_24 ; OWSPRITE_CLERK
-	dw TILESET_141, SPRITE_ANIM_7E, OWFRAMEGROUP_25 ; OWSPRITE_GRCLERK
-	dw TILESET_142, SPRITE_ANIM_7F, OWFRAMEGROUP_26 ; OWSPRITE_65
-	dw TILESET_143, SPRITE_ANIM_80, OWFRAMEGROUP_27 ; OWSPRITE_66
-	dw TILESET_144, SPRITE_ANIM_81, OWFRAMEGROUP_28 ; OWSPRITE_67
-	dw TILESET_145, SPRITE_ANIM_82, OWFRAMEGROUP_29 ; OWSPRITE_68
-	dw TILESET_146, SPRITE_ANIM_83, OWFRAMEGROUP_2A ; OWSPRITE_GRSHIP
-	dw TILESET_147, SPRITE_ANIM_84, OWFRAMEGROUP_2B ; OWSPRITE_GRSHIPBEAM
-	dw TILESET_001, SPRITE_ANIM_86, OWFRAMEGROUP_00 ; OWSPRITE_VULCANO_SMOKE_TCG
-	dw TILESET_001, SPRITE_ANIM_86, OWFRAMEGROUP_01 ; OWSPRITE_CURSOR_TCG
-	dw TILESET_003, SPRITE_ANIM_87, OWFRAMEGROUP_02 ; OWSPRITE_6D
-	dw TILESET_003, SPRITE_ANIM_88, OWFRAMEGROUP_03 ; OWSPRITE_6E
-	dw TILESET_003, SPRITE_ANIM_89, OWFRAMEGROUP_04 ; OWSPRITE_6F
-	dw TILESET_003, SPRITE_ANIM_8A, OWFRAMEGROUP_05 ; OWSPRITE_70
-	dw TILESET_077, SPRITE_ANIM_8D, OWFRAMEGROUP_0A ; OWSPRITE_71
-	dw TILESET_077, SPRITE_ANIM_8D, OWFRAMEGROUP_0B ; OWSPRITE_72
-	dw TILESET_078, SPRITE_ANIM_8E, OWFRAMEGROUP_0C ; OWSPRITE_73
-	dw TILESET_07A, SPRITE_ANIM_90, OWFRAMEGROUP_0D ; OWSPRITE_74
-	dw TILESET_07B, SPRITE_ANIM_8B, OWFRAMEGROUP_06 ; OWSPRITE_75
-	dw TILESET_07B, SPRITE_ANIM_8B, OWFRAMEGROUP_07 ; OWSPRITE_76
-	dw TILESET_07C, SPRITE_ANIM_8C, OWFRAMEGROUP_08 ; OWSPRITE_77
-	dw TILESET_07C, SPRITE_ANIM_8C, OWFRAMEGROUP_09 ; OWSPRITE_78
+	dw TILESET_MARK, SPRITE_ANIM_67, OWFRAMEGROUP_0E            ; $000
```

Comment:
Personally I feared redundancy while updating this section, since the numbers would have replaced with the already stated character/object name. So should we go with "OWSPRITE_##" or "OWSPRITE_NAME"?

Reply:
They should be the same as the constant names, so that when someone does a Ctrl+F on the constant, they know these entries are associated with it

Reply:
Well looks like I forgot to actually update those constants, so that explains the confusion :/
Should be all good now.